docs/conf.py

```diff
@@ -1,7 +1,5 @@
-# -*- coding: utf-8 -*-
-#
 # pyrlp documentation build configuration file, created by
-# sphinx-quickstart on Mon Feb 16 14:20:49 2015.
+# sphinx-quickstart on Thu Oct 16 20:43:24 2014.
```

Comment:
This is a weird change, but probably doesn't matter.

Reply:
may be coming from the template commit?